contract/scripts/wait-for-container-ready.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+cd demo
+
+timeout 300 bash -c '
+TARGET_HEIGHT=1111
```

Comment:
should `TARGET_HEIGHT` be hard coded into the script? would an env variable be more flexible?